misc/io.github.Tremulous.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Daniel Rusek <mail@asciiwolf.com> -->
+<component type="desktop-application">
+  <id>com.grangerhub.Tremulous</id>
```

Comment:
The app id is incorrect. It should be `io.github.Tremulous`. Why didn't you use the latest file from my [Flathub PR](https://github.com/flathub/flathub/pull/5290/files)?

Reply:
> The app id is incorrect. It should be `io.github.Tremulous`. Why didn't you use the latest file from my [Flathub PR](https://github.com/flathub/flathub/pull/5290/files)?

Oopsie! Pushed the correct files, but I also want to update the screenshots.